src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -328,7 +333,7 @@ exports.createFiberFromText = function(content : string, priorityLevel : Priorit
   return fiber;
 };
 
-function createFiberFromElementType(type : mixed, key : null | string) : Fiber {
+function createFiberFromElementType(type : mixed, key : null | string, debugOwner : null | Fiber | ReactInstance) : Fiber {
```

Comment:
I didn't really try anything else. 

>create something like getDebugFiberWorkInProgress

The problem here is that the fiber isn't created yet so you wouldn't be able to set a reference.